src/components/Thread/Sendbar.vue

```diff
@@ -430,6 +464,10 @@ body.dark {
         background: url("../../assets/images/ic_attach_white.png") no-repeat;
         background-size: cover;
     }
+    #templates {
+        background: url('../../assets/images/vector/templates-dark.svg') no-repeat;
+        background-size: cover;
+    }
```

Comment:
I'm not crazy about this button location - it interrupts more commonly used features. 

I think a popout bar like slack's text format menu would do better for secondary features. (Perhaps pop above, rather than below)

<img width="609" alt="Screen Shot 2021-01-12 at 9 02 54 PM" src="https://user-images.githubusercontent.com/1234465/104408734-8d69da00-5519-11eb-909c-3dd526e61d8e.png">

Reply:
For initial release, we are keeping it like this. With more functions to come, we definitely are going to follow what you have suggested like Slack.